### ISSUE TITLE
Some feature requests for HTML backend

### ISSUE THREAD

**Issue by bkamins** [MakieOrg/Makie.jl#2326]
Would it be possible to have the following interaction features supported by stand alone HTML plots (this is similar to what plotly provides and is really important when shipping stand alone HTML to end-users):
- zooming in/out
- shifting axis
- turning on/off lines (by clicking on the corresponding legend or line)
- showing data point information by hovering on data points (lower priority)

Thank you!

**Comment by SimonDanisch** [comment]
Yeah, these are definitely all planned but it's not easy to fund the work on it right now!
